pkg/controller/migplan/validation.go

```diff
@@ -81,16 +82,15 @@ const (
 
 // Reasons
 const (
-	NotSet        = "NotSet"
-	NotFound      = "NotFound"
-	KeyNotFound   = "KeyNotFound"
-	NotDistinct   = "NotDistinct"
-	LimitExceeded = "LimitExceeded"
-	NotDone       = "NotDone"
-	Done          = "Done"
-	Conflict      = "Conflict"
-	NotHealthy    = "NotHealthy"
-	Incorrect     = "Incorrect"
```

Comment:
is this intentional?

Reply:
Yes the Incorrect const wasn't used before.